src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -390,7 +391,7 @@ def load_sub_model(
     return loaded_sub_model
 
 
-class DiffusionPipeline(ConfigMixin):
+class DiffusionPipeline(ConfigMixin, TextualInversionLoaderMixin):
```

Comment:
```suggestion
class DiffusionPipeline(ConfigMixin):
```

textual inversion does not apply to all pipelines (e.g. we have audio pipelines now) - let's make sure to only add it to relevant pipelines to begin with